fmc/src/flow/rt_alias.rs

```diff
@@ -11,33 +11,63 @@ Abstract:
     Crypto helper routines
 
 --*/
-
 use crate::flow::dice::{DiceInput, DiceLayer, DiceOutput};
+use crate::flow::pcr::{extend_current_pcr, extend_journey_pcr};
 use crate::fmc_env::FmcEnv;
+use crate::HandOff;
+use caliptra_common::cprintln;
 use caliptra_drivers::CaliptraResult;
 
 #[derive(Default)]
 pub struct RtAliasLayer {}
 
 impl DiceLayer for RtAliasLayer {
     /// Perform derivations for the DICE layer
-    fn derive(env: &FmcEnv, _: &DiceInput) -> CaliptraResult<DiceOutput> {
-        Self::extend_pcrs(env)?;
-        Err(0)
+    fn derive(env: &FmcEnv, _input: &DiceInput) -> CaliptraResult<DiceOutput> {
+        // At this point PCR0 & PCR1 must have the same value. We use the value
+        // of PCR1 as the data for deriving the CDI
+        let _measurement = env
+            .pcr_bank()
+            .map(|p| p.read_pcr(caliptra_drivers::PcrId::PcrId1));
+
+        // TODO : implement derivation.
+        // Derive the Rt layer CDI.
+        //let cdi = Self::derive_cdi(env, _measurement, input.cdi)?;
+        Err(0xdead)
```

Comment:
If this gets merged will FMC fail to boot? Or is this code not linked yet?

Reply:
It is not not being invoked.  Our integration test that jumps from ROM to FMC to RT is successfully running on CI as expected.

Reply:
@jhand2 , does the current/journey PCR allocation match your expectation ?